lib/utils/__tests__/isStandardSyntaxAtRule.test.js

```diff
@@ -1,136 +1,112 @@
 	it('non nested at-rules without quotes', () => {
-		atRules('@charset UTF-8;', (atRule) => {
-			expect(isStandardSyntaxAtRule(atRule)).toBeTruthy();
-		});
+		expect(isStandardSyntaxAtRule(atRules('@charset UTF-8;')[0])).toBeTruthy();
 	});
 
 	it("non nested at-rules with `'` quotes", () => {
-		atRules("@charset 'UTF-8';", (atRule) => {
-			expect(isStandardSyntaxAtRule(atRule)).toBeTruthy();
-		});
+		expect(isStandardSyntaxAtRule(atRules("@charset 'UTF-8';")[0])).toBeTruthy();
 	});
 
 	it('non nested at-rules with `"` quotes', () => {
-		atRules('@charset "UTF-8";', (atRule) => {
-			expect(isStandardSyntaxAtRule(atRule)).toBeTruthy();
-		});
+		expect(isStandardSyntaxAtRule(atRules('@charset "UTF-8";')[0])).toBeTruthy();
 	});
 
 	it("non nested at-rules with `'` quotes and without space after name", () => {
-		atRules("@charset'UTF-8';", (atRule) => {
-			expect(isStandardSyntaxAtRule(atRule)).toBeTruthy();
-		});
+		expect(isStandardSyntaxAtRule(atRules("@charset'UTF-8';")[0])).toBeTruthy();
 	});
 
 	it('non nested at-rules with `"` quotes and without space after name', () => {
-		atRules('@charset"UTF-8";', (atRule) => {
-			expect(isStandardSyntaxAtRule(atRule)).toBeTruthy();
-		});
+		expect(isStandardSyntaxAtRule(atRules('@charset"UTF-8";')[0])).toBeTruthy();
 	});
 
 	it('non nested at-rules with function and without space after name', () => {
-		atRules('@import url("fineprint.css") print;', (atRule) => {
-			expect(isStandardSyntaxAtRule(atRule)).toBeTruthy();
-		});
+		expect(isStandardSyntaxAtRule(atRules('@import url("fineprint.css") print;')[0])).toBeTruthy();
 	});
 
 	it('nested at-rules', () => {
-		atRules('@media (min-width: 100px) {};', (atRule) => {
-			expect(isStandardSyntaxAtRule(atRule)).toBeTruthy();
-		});
+		expect(isStandardSyntaxAtRule(atRules('@media (min-width: 100px) {};')[0])).toBeTruthy();
 	});
 
 	it('nested at-rules with newline after name', () => {
-		atRules('@media\n(min-width: 100px) {};', (atRule) => {
-			expect(isStandardSyntaxAtRule(atRule)).toBeTruthy();
-		});
+		expect(isStandardSyntaxAtRule(atRules('@media\n(min-width: 100px) {};')[0])).toBeTruthy();
 	});
 
 	it('nested at-rules with windows newline after name', () => {
-		atRules('@media\r\n(min-width: 100px) {};', (atRule) => {
-			expect(isStandardSyntaxAtRule(atRule)).toBeTruthy();
-		});
+		expect(isStandardSyntaxAtRule(atRules('@media\r\n(min-width: 100px) {};')[0])).toBeTruthy();
 	});
 
 	it('nested at-rules without space after name', () => {
-		atRules('@media(min-width: 100px) {};', (atRule) => {
-			expect(isStandardSyntaxAtRule(atRule)).toBeTruthy();
-		});
+		expect(isStandardSyntaxAtRule(atRules('@media(min-width: 100px) {};')[0])).toBeTruthy();
 	});
 
-	it('ignore `@content` inside mixins newline', () => {
-		const sass = '@mixin mixin()\n  @content';
+	// eslint-disable-next-line jest/no-disabled-tests -- TODO: `postcss-sass` parser does not support `@mixin`.
+	it.skip('ignore `@content` inside mixins newline', () => {
```

Comment:
[note] This skip does not affect on the coverage:

---

<img width="640" alt="image" src="https://user-images.githubusercontent.com/473530/94292574-1c1d1000-ff98-11ea-8eb8-1ee551335597.png">

Reply:
See also https://coveralls.io/builds/33727095/source?filename=lib/utils/isStandardSyntaxAtRule.js

Reply:
Has anybody left an issue on `postcss-sass` about this? Maybe we can submit a fix upstream as well.

Reply:
I found a PR to add the support of `@mixin` at-rule, but it seems WIP (and too old). 😢 

https://github.com/AleshaOleg/postcss-sass/pull/56